frontend/src/components/widgets/NumberInput/styled-components.ts

```diff
@@ -44,14 +44,13 @@ export const StyledInputControl = styled.button(({ theme }) => ({
   border: "none",
   height: theme.sizes.full,
   display: "flex",
-  cursor: "pointer",
```

Comment:
This seems unnecessary - the cursor is always a pointer when over an _enabled_ button (and an arrow otherwise).